.gitignore

```diff
@@ -12,6 +12,7 @@ html_docs
 build
 reports
 coverage
+**/benchmarks/*.json
```

Comment:
We wouldn't need this if we separate the reports from the actual benchmarks!

Reply:
Reason i pushed it here is because there is no reports directory in core and we have to create it manually then before the bechmark script runs. Json reporter does create folders, only creates files in existing dir.

Reply:
Do you want to manually create the folder and use it or this you are okay with the above change?

Reply:
We decided to put the generated benchmark results in the reports folder.